.releaserc

```diff
@@ -18,4 +18,10 @@ plugins:
   - preset: conventionalcommits
 - - "@semantic-release/release-notes-generator"
   - preset: conventionalcommits
-- "@semantic-release/github"
+- - "@semantic-release/exec"
+  - prepareCmd: "VERSION=${nextRelease.version} npm run bump:version"
+- - "@semantic-release/git"
+  - assets:
+    - "action.yml"
+    message: "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
```

Comment:
is this going to do a commit or PR? cause commit will not work as commits directly to master are blocked

Reply:
It does a direct commit as making a PR won't work if we want to include it in release.

Reply:
so can be removed